app/assets/stylesheets/layout/_signup.scss

```diff
@@ -0,0 +1,42 @@
+#signup{
+  background-image: url("/assets/image-5.jpg");
```

Comment:
it's better to use image-url in your scss assets:
```css
background-image: image-url("image-5.jpg");
```

You can find more information [here], but this is the basics: (http://guides.rubyonrails.org/asset_pipeline.html)
2.3.2 CSS and Sass

When using the asset pipeline, paths to assets must be re-written and sass-rails provides -url and -path helpers (hyphenated in Sass, underscored in Ruby) for the following asset classes: image, font, video, audio, JavaScript and stylesheet.

image-url("rails.png") becomes url(/assets/rails.png)
image-path("rails.png") becomes "/assets/rails.png".